client/src/components/Donation/PaypalButton.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { PaypalButton } from './PaypalButton';
+
+const commonProps = {
+  donationAmount: 500,
+  donationDuration: 'month',
+  handleProcessing: () => null,
+  isDonating: false,
+  onDonationStateChange: () => null
+};
+
+const donationData = {
+  redirecting: false,
+  processing: false,
+  success: false,
+  error: null
+};
+
+interface Ref {
```

Comment:
You should not need to define a `Ref` interface. This should come directly from React: [Example](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/components/Header/components/universal-nav.tsx#L7)

Reply:
Also, seeing as this is a new file, shall we change this to kebab-case?

Reply:
Thanks, both requested changes are added.